### PR TITLE
add Sorcia

### PR DESCRIPTION
**Pull request by mysticmode**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [ ] Your submissions are formatted according to the following requirements:
        
``- [Name](http://homepage/) - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``

Depends on proprietary services:

``- [Name](http://homepage/) `⚠` - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``

Non-free software in `non-free.md`:

``- [Name](http://homepage/) `⊘ Proprietary` - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `Language` ``

- [ ] Your additions are ordered alphabetically.
- [ ] Your additions are [Free software](https://en.wikipedia.org/wiki/Free_software), or if not they have been added to [non-free](non-free.md) and marked `⊘ Proprietary`.
- [ ] If your additions depend on proprietary services outside the user's control, they must be marked `⚠`.
- [ ] Your additions are not already listed at [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin) (IT infrastructure management), [staticgen.com](https://www.staticgen.com/) or [staticsitegenerators.net](https://staticsitegenerators.net/) (static site generators).
- [ ] Your additions have their `License` main server-side `Language`/platform/requirement listed.
- [ ] Any licenses you have added are in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [ ] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [ ] If the main link of your addition points to the program's source code, there is no need to add a duplicate _Source Code_ link. The _Demo_ and _Clients_ links are not mandatory.
- [ ] Any category you are creating has the minimum requirement of 3 items. If not, your addition may be inserted into `Misc/Other`.
- [ ] Any software project you are adding to the list is actively maintained.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1314,6 +1314,7 @@ _See also [Ticketing](#ticketing), [Task management/To-do lists](#task-managemen
 - [Redmine](http://www.redmine.org/) - Redmine is a flexible project management web application. ([Demo](http://demo.redmine.org/), [Source Code](https://svn.redmine.org/redmine/)) `GPL-2.0` `Ruby`
 - [RhodeCode](https://rhodecode.com/) - RhodeCode is an open source platform for software development teams. It unifies and simplifies repository management for Git, Subversion, and Mercurial. ([Demo](https://try.rhodecode.com/), [Source Code](https://code.rhodecode.com/)) `AGPL-3.0` `Python`
 - [SCM Manager](https://www.scm-manager.org/) - The easiest way to share and manage your Git, Mercurial and Subversion repositories over http. ([Source Code](https://bitbucket.org/sdorra/scm-manager/src)) `BSD-3-Clause` `Java`
+- [sorcia](https://sorcia.org) - Sorcia is a minimalist web frontend for Git repositories with fine-grained user access controls, multiple themes to choose based on your taste and many more. ([Source Code](https://git.mysticmode.org/r/sorcia)) `AGPL-3.0` `Go`
```
Comment (SuperSandro2000) [inline README.md:1317]:
```suggestion
- [sorcia](https://sorcia.org) - Sorcia is a minimalist web frontend for Git repositories with fine-grained user access controls, multiple themes to choose based on your taste and much more. ([Source Code](https://git.mysticmode.org/r/sorcia)) `AGPL-3.0` `Go`
```

Or are there very many themes?

Comment (mysticmode) [reply README.md:1317]: No, there are only three themes at the moment. I should have worded it differently, when I say much more, it means more features that are there.

I'm in the process of implementing mailing lists into Sorcia and few others which I don't want to share it here at the moment. So, I though that "much more" would be appropriate.

Anyways, it doesn't matter much, I'll go with what you have suggested.

Comment (mysticmode) [reply README.md:1317]: Thing is - your approval process it too laggy! And I though I don't want to come back for changes.

Comment (SuperSandro2000) [reply README.md:1317]: I just pointed out a grammatical mistake.

Comment (mysticmode) [reply README.md:1317]: Yes, I didn't take that offensive. I just think that the approval process is taking more time and I notified that.

As I said in the first message, I'm fine with your changes - we can go with that, if it is Okay with other members of this repository.

**Pull request merged**